### DISCUSSION TITLE
Action button feature

### DISCUSSION THREAD

**Discussion by ghost** [laravel/nova-issues#5413] [Ideas & Feature Requests]
The "issue" with actions is that they are "hidden". Most of the time that is perfectly fine, but there are scenarios when an important and context-specific action should be made more noticeable (e.g. as the primary or secondary button).

"Action button" feature could solve this issue. It should provide the same set of features as original actions, but also allow to display the button on index and detail screens, in an emphasised manner.

**Comment by ghost** [comment]
Bump on this one. This is still very much welcome feature. :)

**Comment by milewski** [comment]
Would something like this https://github.com/dcasia/icon-action-toolbar help?

**Reply by ghost** [reply]
Hi @milewski. This is pretty close to what I'm looking for.

Is it possible to:
1. Register those custom actions on detail screens?
2. Display a label next to the icon to make crystal clear for the user what an action is about?
3. Change color of an icon to emphasize that one of them is more important than others?

Thanks!

**Reply by milewski** [reply]
1 and 3 yeah currently possible, 2. I can add support for it should be pretty easy / straight forward

**Reply by milewski** [reply]
done: https://github.com/dcasia/icon-action-toolbar/pull/1

![image](https://github.com/laravel/nova-issues/assets/2874967/1e331bce-33dc-491d-af46-1592a7b4c585)

**Reply by ghost** [reply]
Nice! I'll give it a try.

**Reply by bkuhl** [reply]
Oh wow, this looks pretty awesome!  You all have a lot of great additions to Nova, will definitely check it out :)
